Managers/TimerMan.h

```diff
@@ -238,7 +240,10 @@ namespace RTE {
 		float m_SimSpeed; //!< The simulation speed over real time.
 		float m_TimeScale; //!< The relationship between the real world actual time and the simulation time. A value of 2.0 means simulation runs twice as fast as normal, as perceived by a player.
 
-		bool m_SimPaused; //!< Simulation paused; no real time ticks will go to the sim accumulator.
+		// Note - these are atomic to allow other threads to pause the simulation (which also resets the SimAccumulator).
+		std::atomic<bool> m_SimPaused; //!< Simulation paused; no real time ticks will go to the sim accumulator.
```

Comment:
Yeah, we can undo (most of) them. Some of these are genuine bugfixes that are good to keep, but the std::atomic isn't necessary anymore. It's not particularly useful because you need to wait till the very end of the tick for the update to pause anyways to be able to read the simulation safely.

Reply:
Makes sense, I'll undo it then, but I guess keep the sim accumulator zeroing when pausing?

Reply:
Yep!